tests/e2e/pageObjects/browser-page.ts

```diff
@@ -972,6 +980,17 @@ export class BrowserPage {
             await t.expect(scannedResults).gt(rememberedScanResults, { timeout: 3000 });
         }
     }
+
+    /**
+     * Open Select Index droprown and select option
+     * @param index The name of format
+     */
+    async selectIndexByName(index: string): Promise<void> {
+        const option = Selector(`[data-test-subj="mode-option-type-${index}"]`);
```

Comment:
I think the `"` are redundant

Reply:
no, without `"` it doesn't work, I checked